eahub/templates/eahub/privacy_policy.html

```diff
@@ -1,9 +1,15 @@
-{% extends 'base.html' %}
+{% extends 'base_bs5.html' %}
 
 {% block title %}Privacy Policy{% endblock %}
 
+<!--navbar block required include after bs3 to bs5 upgrade for page to format correctly-->
```

Comment:
The navbar and the footer still look on bs3. It should look like on the About page: https://eahub.org/about/